indico/web/client/styles/partials/_buttons.scss

```diff
@@ -289,7 +289,7 @@ a.i-big-button {
 
 .i-button.signin {
```

Comment:
you can remove this whole definition since the button no longer exists. it was from the legacy login page (59afb5b37) which no longer exists